src/backbone-marionette.js

```diff
@@ -0,0 +1,111 @@
+var Marionette;
+
+import
+  {
+    bindEntityEvents,
+    unbindEntityEvents,
+    proxyBindEntityEvents,
+    proxyUnbindEntityEvents
+  } from './bind-entity-events';
+
+import
+  {
+    proxyRadioHandlers,
+    unproxyRadioHandlers
+  } from './radio-helpers';
+
+import
+  {
+    triggerMethod,
+    triggerMethodOn,
+    triggerMethodMany
+  } from './trigger-method';
+
+import extend             from './utils/extend';
+import isNodeAttached     from './utils/isNodeAttached';
+import mergeOptions       from './utils/mergeOptions';
+import getOption          from './utils/getOption';
+import proxyGetOption     from './utils/proxyGetOption';
+import normalizeMethods   from './utils/normalizeMethods';
+import normalizeUIString  from './utils/normalizeUIString';
+import normalizeUIKeys    from './utils/normalizeUIKeys';
+import normalizeUIValues  from './utils/normalizeUIValues';
+import actAsCollection    from './utils/actAsCollection';
+import deprecate          from './utils/deprecate';
+
+import MonitorDOMRefresh from './dom-refresh';
+import MNObject      from './object';
+import Renderer      from './renderer';
+import TemplateCache from './template-cache';
+import AbstractView  from './abstract-view';
+import View          from './view';
+import CollectionView from './collection-view';
+import CompositeView from './composite-view';
+import Behavior      from './behavior';
+import Behaviors     from './behaviors';
+import Region        from './region';
+import RegionManager from './region-manager';
+
+import Application   from './application';
+import AppRouter     from './app-router';
+
+import { enableMarionette } from './metal';
+
+import { FEATURES, isEnabled, setEnabled } from './features';
+
+import MarionetteError from './error';
+
+Marionette = Marionette || {};
+
+// extend Marionette in the meantime
+Object.assign(Marionette, {
+  bindEntityEvents,
+  unbindEntityEvents,
+  proxyBindEntityEvents,
+  proxyUnbindEntityEvents,
+  proxyRadioHandlers,
+  unproxyRadioHandlers,
+  extend,
+  isNodeAttached,
+  mergeOptions,
+  getOption,
+  proxyGetOption,
+  normalizeMethods,
+  normalizeUIString,
+  normalizeUIKeys,
+  normalizeUIValues,
+  actAsCollection,
+  deprecate,
+  MonitorDOMRefresh,
+  triggerMethod,
+  triggerMethodOn,
+  triggerMethodMany,
+  MarionetteError,
+  FEATURES,
+  isEnabled,
+  setEnabled,
+  Application,
+  AppRouter,
+  MonitorDOMRefresh,
+  MNObject,
```

Comment:
@jasonLaster we forgot to remove this, in favor of [line 101](https://github.com/marionettejs/backbone.marionette/pull/2769/files#diff-505d5912b9c98bf87e95e162f9390b4fR101).